README.md

```diff
@@ -1,235 +1,436 @@
-  - Windows: [Git for Windows](https://gitforwindows.org/) or an alternative Git distribution.
-- Install Protobuf (required for gRPC)
-  - Linux: `sudo apt install protobuf-compiler libprotobuf-dev`
-  - Windows: [protoc-21.10-win64.zip](https://github.com/protocolbuffers/protobuf/releases/download/v21.10/protoc-21.10-win64.zip) and add `bin` directory to `Path`
-  - MacOS: `brew install protobuf`
-- Install the [clang toolchain](https://clang.llvm.org/) (required for RocksDB and WASM `secp256k1` builds)
-  - Linux: `apt-get install clang-format clang-tidy clang-tools clang clangd libc++-dev libc++1 libc++abi-dev libc++abi1 libclang-dev libclang1 liblldb-dev libllvm-ocaml-dev libomp-dev libomp5 lld lldb llvm-dev llvm-runtime llvm python3-clang`
-  - Windows: Please see [Installing clang toolchain on Windows](#installing-clang-toolchain-on-windows)
-  - MacOS: Please see [Installing clang toolchain on MacOS](#installing-clang-toolchain-on-macos)
-- Install the [rust toolchain](https://rustup.rs/)
-- If you already have rust installed, update it by running: `rustup update`
-- Install wasm-pack: `cargo install wasm-pack`
-- Install wasm32 target: `rustup target add wasm32-unknown-unknown`
-- Run the following commands:
 
-```bash
-$ git clone https://github.com/kaspanet/rusty-kaspa
-$ cd rusty-kaspa
-```
+<h1 align="center">Kaspa On Rust</h1>
+
+Welcome to the Rust-based implementation of the Kaspa full-node and its ancillary libraries. This Alpha release serves as a replacement to the established <a href="https://github.com/kaspanet/kaspad">Golang node</a> (once the rust rewrite is completed), introducing developers to the possibilities of Rust in the Kaspa network's context.
+
+Our codebase is an experimental ground for developers aiming to push the boundaries of blockchain scalability and speed. The Kaspa node aims to validate the feasibility of a high-speed blockchain, boasting block intervals of one second with prospects for significant enhancements.
+
+Contributors to this repository will engage with the technical challenges of achieving rapid block times and swift confirmation, moving towards the goal of scaling transaction throughput substantially. While our objectives are ambitious, our focus remains on robust and pragmatic code development.
+
+We invite developers and blockchain enthusiasts to collaborate, test, and optimize our Rust implementation. Each line of code here is an opportunity to contribute to the open-source blockchain movement, shaping a platform designed for scalability and speed without compromising on decentralization.
+
+Your feedback, contributions, and issue reports will be integral to evolving this codebase from its Alpha phase into a mature and reliable node in the Kaspa network.
+
+## Installation
+  <details>
+  <summary>Building on Linux</summary>
+  
+  1. Install  general prerequisites
+
+      ```bash
+      sudo apt install build-essential \

```

Comment:
Please have this be a single line. In desktop, it looks like too much whitespace on the right when it's split.

Reply:
+1